snuba/subscriptions/scheduler_consumer.py

```diff
@@ -41,7 +41,7 @@ class MessageDetails(NamedTuple):
     orig_message_ts: float
     # The timestamp the message was first received by Sentry (Relay)
     # It is optional since it is not currently present on all topics
-    received_ts: Optional[float]
+    received_p99: Optional[float]
```

Comment:
maybe im missing context but why the change from `received_ts` to `received_p99`?

Reply:
Naming consistency. This is called received_p99 everywhere. Also, this was never used before and was introduced only in a recent commit.